src/components/side_panel/chart/combo_chart/combo_chart_config_panel.ts

```diff
@@ -0,0 +1,22 @@
+import { _t } from "../../../../translation";
+import { LineBarPieConfigPanel } from "../line_bar_pie_panel/config_panel";
+
+export class ComboConfigPanel extends LineBarPieConfigPanel {
```

Comment:
```suggestion
export class ComboChartConfigPanel extends LineBarPieConfigPanel {
```
To be consistent with `ComboChartDesignPanel`